packages/google_identity_services_web/example/integration_test/js_interop_id_test.dart

```diff
@@ -31,6 +31,27 @@ void main() async {
     });
   });
 
+  group('initialize', () {
+    testWidgets('can receive `use_fedcm_for_prompt` boolean', (_) async {
+      utils.expectFedCMConfig();
+
+      expect(() {
+        id.initialize(IdConfiguration(client_id: 'testing_1-2-3'));
+      }, throwsA(isA<String>()));
```

Comment:
This is not testing the real behavior of `initialize`, but that our mock pukes if we don't pass the [`use_fedcm_for_prompt`](https://developers.google.com/identity/gsi/web/reference/js-reference#use_fedcm_for_prompt) configuration value.

(Another test might have seen to just extract `use_fedcm_for_prompt` from a freshly created `IdConfiguration` object, without changing the behavior of the mock (?)

Reply:
Can you just add a comment to this `expect` explaining that in the code, since it's a bit mysterious otherwise?

Reply:
I will change this test a little bit, so we can assert that constructing a `IdConfiguration` results in a JS Object that has the expected properties, so I can test all of them, rather than doing this weird thing.